### PR TITLE
Support translation

### LINKED ISSUE #16: Support for translation
For now the user facing text is in English:

https://github.com/jupyterlab-contrib/jupyterlab-tour/blob/6dc52f53802d40b5281785f2167fec8c2af48573/src/defaults.tsx#L136-L137

It would be nice to add support for additional languages by using the `ITranslator` service from JupyterLab, so the default tours could also be available in other languages.

### PR DESCRIPTION
**Pull request by fcollonval**
Fixes #16

cc @jtpio

### CODE REVIEW

**Review by jtpio — APPROVED** [review]
Thanks!

**Comment by jtpio** [comment]
Looks like there is a small conflict.

File: src/tourManager.ts [hunk 1]
```diff
@@ -84,6 +98,57 @@ export class TourManager implements ITourManager {
...
+   */
+  addTour(tour: ITour): ITourHandler | null {
+    try {
+      let trans = this._trans;
+      if (tour.translation) {
+        trans = this._translator.load(tour.translation);
+      }
+
+      const handler = this.createTour(
+        `${USER_PLUGIN_ID}:${tour.id}`,
```
Comment (bollwyvl) [inline src/tourManager.ts:125]: Oops, looks like this maybe shouldn't have snuck in here?

Comment (fcollonval) [reply src/tourManager.ts:125]: Oh yes, thanks for catching it. I'll push a fix

**Pull request merged**